components/cookie/banner/src/index.js

```diff
@@ -94,9 +94,16 @@ class CookieBanner extends Component {
 
   render() {
     const {showCookieBanner} = this.state
+    const {onChangeCookieBanner} = this.props
+
     if (!showCookieBanner || this._getHasAcceptedCookie()) {
+      // CookieBanner is not displayed
+      onChangeCookieBanner && onChangeCookieBanner(false)
```

Comment:
Better if you use a default prop with an empty function instead checking if it exists.